.golangci.yml

```diff
@@ -31,3 +31,7 @@ run:
   skip-files:
     - "zz_generated.*\\.go$"
     - ".*_mock\\.go"
+linters-settings:
+  gocyclo:
+    # minimal code complexity to report, 30 by default (but we recommend 10-20)
+    min-complexity: 32
```

Comment:
This seems very specific? What complexity are we trying to get the linter to accept? (I guess i'll find out in a minute)

Reply:
Min complexity is 30, we were at 29. I added 3 more.

Reply:
I was also skeptical about this. I suggest split the code into smaller functions instead of increasing the complexity limit :smile:

Reply:
The complexity was coming from the main function, breaked it down.